### DISCUSSION TITLE
Ayo do you take requests?

### DISCUSSION THREAD

**Discussion by TheWorksOfBH** [NoahWantsDie/fnf-shits#14] [Ideas]
If so can you do Splitathon V2 from VS Dave and Bambi? Couldn't find a good midi for it.

**Comment by DrakeIsBruh** [comment]
isnt that song a wip? i dont think it could be made into a midi just yet

**Reply by TheWorksOfBH** [reply]
I mean this https://youtu.be/U2BS7aCTc5o
